### DISCUSSION TITLE
LTSP proxmox

### DISCUSSION THREAD

**Discussion by Squialz** [ltsp/ltsp#719] [Q&A]
hello i am contacting you today because for my ltsp server i need to be able to send the image to vms and physical pc my server and my vms are on proxmox and i don't know how to connect a pc so that it can booted on the server. should I add a network card on the server to which I will connect my pc or other? and do you have to change parameters in the dns of the server ?

**Comment by alkisg** [comment]
Use a server VM with one NIC in bridged mode.
Follow the default installation instructions: https://ltsp.org/docs/installation/
That's all. You should be able to boot the clients, either physical or VM ones (in bridged mode again) without configuring anything in pfsense.

Regarding DHCP, you may enable the DHCP server in pfsense, but do not set any DHCP boot parameters there, so that the proxydhcp mode of LTSP will work.
